### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jfy133** [comment]
How did you do this @LilyAnderssonLee ? With nf-core tools or manually?

**Comment by LilyAnderssonLee** [comment]
@jfy133 Manually. I got an error by running `nf-core bump-version 1.1.2dev`
image

**Comment by jfy133** [comment]
Ok well you should always resolve the error rather than reverting to manual 😬

1. You're missing the manifest update in nextflow.config
2. The string in the multiQC config should just be `/dev/ ` in the url

**Comment by jfy133** [comment]
And we don't bump the version in the workflow diagram for Dev versions

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.2dev - Augmented Akita Patch [2023-10-11]
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## dev - [unreleased]
```

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/taxprofiler/releases/tag/1.1.1" target="_blank">nf-core/taxprofiler</a>
+  This report has been generated by the <a href="https://github.com/nf-core/taxprofiler/releases/tag/1.1.2dev" target="_blank">nf-core/taxprofiler</a>
```
Comment (jfy133) [inline assets/multiqc_config.yml:2]:
```suggestion
  This report has been generated by the <a href="https://github.com/nf-core/taxprofiler/releases/tag/dev" target="_blank">nf-core/taxprofiler</a>
```

File: assets/multiqc_config.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/taxprofiler/1.1.1/docs/output" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/taxprofiler/1.1.2dev/docs/output" target="_blank">documentation</a>.
```
Comment (jfy133) [inline assets/multiqc_config.yml:4]:
```suggestion
  <a href="https://nf-co.re/taxprofiler/dev/docs/output" target="_blank">documentation</a>.
```

File: docs/images/taxprofiler_logo.svg [hunk 4]
```diff
@@ -7,7 +7,7 @@
    viewBox="0 0 210 297"
    version="1.1"
    id="svg5"
-   inkscape:version="1.1.1 (c3084ef, 2021-09-22)"
+   inkscape:version="1.1.2dev (c3084ef, 2021-09-22)"
```
Comment (jfy133) [inline docs/images/taxprofiler_logo.svg:10]:
```suggestion
   inkscape:version="1.1.1 (c3084ef, 2021-09-22)"
```

**Review by jfy133 — APPROVED** [review]

**Comment by LilyAnderssonLee** [comment]
Thanks @jfy133  
Sorry for a late reply. I am busy with clinical related things these days. 
I agree with you that I should address the issues rather than manual corrections.

**Comment by jfy133** [comment]
No worries!

**Pull request merged**